src/device-registry/utils/create-site.js

```diff
@@ -628,6 +628,7 @@ const createSite = {
           ...body,
           ...roadResponseData,
           ...altitudeResponseData,
+          data_provider: constants.DATA_PROVIDER_MAPPINGS(body.network),
```

Comment:
Hi @BenjaminSsempala , this is good work! Is this body.network always defined in this process? If no, what happens in case` body.network === "undefined"`?

Reply:
Hey @Baalmart 
If the `body.network is undefined`,  the switch statement will not match any of the defined cases ("airqo" or "usembassy") and will fall back to the default case, which is "airqo".

Reply:
thanks @BenjaminSsempala